r-getting-started.Rmd

```diff
@@ -0,0 +1,329 @@
+using the second search box that reads "Find in Topic."
+In this case, type "decimal" to find information about how the function
+determines the number of decimal points to print in the output.
+
+Luckily, R help documentation tends to be formatted very consistently. At the
+very top, you'll see the name of the function ("Round", which in this case
+represents a collection of related functions) followed by the name of its
+package inside curly brackets ("{base}", this is important because sometimes
+different packages have functions with the same names!). Below that, a short title
+indicates the purpose of the function. "Description" is a more extensive
+explanation that should help you figure out if the function is appropriate for
+your use. "Usage" provides information on how the function should be applied
+through code, and "Arguments" is like a legend for the code described in Usage.
+The last subheadings, "Details" through "References" and "Examples", should be
+self-explanatory.
+
+When you're writing code in your script, you can also access help documentation
+by prefacing a function with a question mark:
+
+```{r eval=FALSE}
+?sum
+```
+
+After executing that code, you should see the help page for that function appear
+in the lower right pane.
+
+RStudio also provides helpful pop-up windows that attempt to predict the
+function name you're typing. You may see windows that appear as you're coding,
+which include short versions of the help documentation.
+
+## Getting Help Online {#r-getting-started-web-help}
+
+Sometimes you may not be able to find the answers you seek in the
+documentation available through RStudio.
+This often happens when you need to do something
+completely new and you don't know where to start. In that scenario, there are
+multiple internet resources that are helpful:
+
+* blogs
+* Twitter
+* tutorials
+* question pages on Stack Overflow, an open forum that allows community members
+to ask and answer questions
+
+It can be challenging to find the right words to search when you're
+troubleshooting something new. Don't be discouraged, and think about different
+ways you can apply the terminology you're learning to make your search terms
+clearer and more specific.
+
+Even when you find information that seems useful,
+these resources can be overwhelming and confusing.
+Some things to look for in a Stack Overflow discussion include:
+
+* Is the question clearly stated and is there example code? If you're
+debugging, does the example code look like yours?
+* Answers have upvotes and downvotes. Is there one clear answer that has a lot
+of upvotes in response to a question?
+* How complex is the answer? While some questions will necessarily have a
+complicated answer, for many common programming tasks, there should be a
+solution that only takes a few lines of code.
+* Do you recognize any terms in the solution? If you don't, are there other
+terms for which you could search?
+
+
+> Conventions
```

Comment:
```suggestion
> **Conventions**:
```

Reply:
This was included in the python materials as well, in case we are being specific about parallel formatting and whatnot.